src/plugins/geomap/demo/config-fr.js

```diff
@@ -138,7 +147,7 @@ var wet_boew_geomap = {
 						"AgGXbm1wuz7VF85YqIHXPsTjW6lu3dpmfBvot2y5ZCkkJxbE2pW6cFCItLVWeUrEpr9erUuUse2Gl9CZfnrsYfwFqWYQCxROHDQAAAABJRU5ErkJggg==",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de sables bitumineux"
+						name: "Oil sands mines"
```

Comment:
Do we need to use the English name? Can we keep the French name?

Reply:
Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448